Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,175 @@
+
+                float BMI = CalculateBmi(weight, height / 100);
+
+                Console.WriteLine("Your BMI is " + BMI);
+
+                WeightType(BMI);
+
+                calculateAgain = ResponseValidation();
+            }
+
+            Console.WriteLine("Exiting...");
+
+        }
+
+        public static string PromptString(string message)
+        {
+            Console.Write(message);
+            var input = Console.ReadLine();
+
+            if (string.IsNullOrEmpty(input))
+            {
+                Console.Write(Environment.NewLine + "Name cannot be empty.");
+                return "-";
+            }
+
+            Console.Write(Environment.NewLine);
+            return input;
+        }
+
+        public static int PromptInt(string message)
+        {
+            Console.Write(message);
+            var input = Console.ReadLine();
+
+            if (string.IsNullOrEmpty(input))
+            {
+                Console.Write(Environment.NewLine);
+                return 0;
+            }
+
+            if(!int.TryParse(input, out int output))
+            {
+                Console.Write(Environment.NewLine + "\"" + input + "\" is not a valid number.");
+                return -1;
+            }
+
+            Console.Write(Environment.NewLine);
+            return output;
+        }
+
+        public static float PromptFloat(string message)
+        {
+            Console.Write(message);
+            var input = Console.ReadLine();
+
+            if (string.IsNullOrEmpty(input))
+            {
+                Console.Write(Environment.NewLine);
+                return 0;
+            }
+
+            if (!float.TryParse(input, out float output))
+            {
+                Console.Write(Environment.NewLine + "\"" + input + "\" is not a valid number.");
+                return -1;
+            }
+
+            Console.Write(Environment.NewLine);
+            return output;
+        }
+
+        public static float CalculateBmi(float weight, float height)
+        {
+            const string failedCalc = "Failed calculating BMI. Reason:";
+            if (weight <= 0 && height <= 0)
+            {
+                Console.WriteLine(failedCalc);
+                Console.WriteLine("Weight cannot be equal or less than zero, but was " + weight + ".");
+                Console.WriteLine("Height cannot be less than zero, but was " + height + ".");
+                return -1;
+            }
+            else if (weight <= 0)
+            {
+                Console.WriteLine(failedCalc);
+                Console.WriteLine("Weight cannot be equal or less than zero, but was " + weight + ".");
+                return -1;
+            }
+            else if (height <= 0)
+            {
+                Console.WriteLine(failedCalc);
+                Console.WriteLine("Height cannot be equal or less than zero, but was " + height + ".");
+                return -1;
+            }
+            //else if (weight >= height)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4885366)